core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -1096,9 +1096,41 @@ export class WorkflowActionService {
     return this.tempWorkflow;
   }
 
+  public setOperatorCustomName(operatorId: string, newDisplayName: string, userFriendlyName: string): void {
```

Comment:
we have three names now 
1. operatorName
2. displayName
3. userFriendlyName.
I think 2 and 3 might have some overlap. Should the `newDisplayName` actually be `newUserFriendlyName`?

Reply:
No, if I understand correctly, `userFriendlyName` is the default name for an operator (stored in `OperatorMetadata`), as opposed to `displayName` which is the actual name to be displayed.

Reply:
Ok